scripts/seshat.conf

```diff
@@ -0,0 +1,3 @@
+#Commenting out for initial Yocto setup
+#[Unit]
+#After=CcspPandMSsp.service CcspMtaAgentSsp.service
```

Comment:
I had left these commented out in order to have an idea of what might need to go in there when getting around to using this file.